### PR TITLE
WIP: render default certificate template in print collector form

### PR DESCRIPTION
**Pull request by pankaj-pant**
## Description

Clearly describe what has been changed. Include relevant context or background.
Explain how the issue was fixed (if applicable) and the root cause.

Link this pull request to the GitHub issue (and optionally name the branch `ocrvs-<issue #>`)

## Checklist

- [ ] I have linked the correct Github issue under "Development"
- [ ] I have tested the changes locally, and written appropriate tests
- [ ] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [ ] I have updated the changelog with this change (if applicable)
- [ ] I have updated the GitHub issue status accordingly

### CODE REVIEW

File: packages/client/src/components/form/FormFieldGenerator.tsx [hunk 1]
```diff
@@ -785,6 +785,11 @@ export const mapFieldsToValues = (
       ...evalParams
     )
 
+    if (field.type === SELECT_WITH_OPTIONS && !field.initialValue) {
+      fieldInitialValue =
+        field.options.find((x) => x.isDefault === true)?.value ?? ''
```
Comment (tareq89) [inline packages/client/src/components/form/FormFieldGenerator.tsx]: Can we make it a bit more clean like this:
```
        field.options.find((x) => x.isDefault)?.value ?? ''
```

File: packages/client/src/components/form/FormFieldGenerator.tsx [hunk 2]
```diff
@@ -1074,76 +1078,79 @@ class FormSectionComponent extends React.Component<Props> {
...
+                            )
+                            const updatedValues = Object.assign(
+                              {},
+                              values,
+                              transformedData[this.props.id]
+                            )
+                            setValues(updatedValues)
+                          }
+                        } as ILoaderButton)
+                      : field
```
Comment (Zangetsu101) [inline packages/client/src/components/form/FormFieldGenerator.tsx]: What changed in this diff? I couldn't find any :sweat_smile:

**Review by Zangetsu101 — APPROVED** [review]

File: packages/client/src/components/form/FormFieldGenerator.tsx (hunk 2 above)
Comment (pankaj-pant) [reply packages/client/src/components/form/FormFieldGenerator.tsx]: Yep, something my formatter probably did which I didn't notice. And it seems Github visualises the diff a bit differently
Screenshot 2025-07-04 at 8 11 48
